kudu/Dockerfile

```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 
 EXPOSE 8181
 
+# Install ClamAV
+RUN apt-get install -y clamav
+RUN freshclam
+
+# Copy custom scripts
+RUN mkdir /custom_scripts
+COPY custom_scripts/* /custom_scripts/
+RUN chmod 755 /custom_scripts/*
```

Comment:
Combine these run commands

Reply:
I ran a few manual tests on this, looks like we first need to copy the folder onto the container then run the chmod 755 for all the files inside that folder to get the permissions. Hence, combining the run command and then running the copy might not work in this case as the files inside /custom_scripts will not get the 755 permissions.